enlighten/Controller.py

```diff
@@ -716,7 +717,11 @@ def default_missing(local_name, empty_value=None, cloud_name=None):
                 default_missing("model", None, "wp_model")
                 default_missing("detector", "iDus")
                 default_missing("serial_number", device.settings.eeprom.detector_serial_number, "wp_serial_number")
+                default_missing("raman_intensity_coeffs", [])
 
+                device.settings.eeprom.raman_intensity_calibration_order = len(device.settings.eeprom.raman_intensity_coeffs) - 1
```

Comment:
some breadcrumbs. These types come from Wasatch.PY.

device: WasatchDeviceWrapper
settings: SpectrometerSettings
eeprom: EEPROM

I'm going to add a comment above this line that points out the type of eeprom here so it doesn't get confused for Enlighten's EEPROMEditor or EEPROMWriter